contents/swift/welcome-to-swift.md

```diff
@@ -144,9 +146,45 @@ Here are some of the benefits of using `guard` statement over `if-let` statement
 1. Unlike the `if-let` statement, using the `guard` statement causes `past` to remain defined and can be used till the function exits.
 1. While using `if-let` statements can lead to deeply nested `if-let` statements (i.e. pyramid of doom), `guard` statements allow us to have the happy path to be not indented, thereby increasing code readability.
 
-## Data Types
+### Benefit: Handle Post-operations easily
+
+The `defer` key word in Swift provides an easy and safe way to execute some code before leaving current scope. It will be helpful if you need to do post-operations in the function which has many return points.
```

Comment:
It is helpful when you need to do post-operations in a function which has many points of return.